### DISCUSSION TITLE
Allow buildings to dim like disabled/EMP'd units do

### DISCUSSION THREAD

**Discussion by ayylmaoRotE** [Phobos-developers/Phobos#874] [Ideas and feedback]
This request is for the extension of the Ares DeactivateDimEMP= tag, or rather create a new one that affects structures. It may be easier to mimic ExtraUnitLight but at a structure level.

This tag would cause the structure's light to be reduced by the value/percentage. 

e.g. the building below is powered=yes and has the following tag, whenever it is power toggled, EMP'd or power is low, its light would be reduced to 50%, similar to EMP'd units and disabled Robot Tanks.

[TechnoType]
**DeactivateBuildingEMP=0.5;** Reduces disabled, EMP'd and/or low powered structure light by 50%

**Comment by yamanekoayf** [comment]
Would be nice if this dimming effect would work on other shp based techno types as well!
